src/api/basics/01-single-video.md

```diff
@@ -9,6 +9,26 @@ video-sources:
   - src: https://storage.googleapis.com/kino-assets/single-video/video.mp4
     type: video/mp4; codecs="avc1.640032,mp4a.40.2"
 thumbnail: https://storage.googleapis.com/kino-assets/single-video/thumbnail.png
+media-session-artwork:
```

Comment:
Do we already have linter warnings so that we don't forget to add media session artwork to future .md files?

Reply:
I can see how that would be useful, but I don't believe I've ever linted front-matter blocks in markdown files. What tool would you recommend for the task?

Reply:
https://www.npmjs.com/package/remark-frontmatter maybe. We use it for web.dev. See https://github.com/GoogleChrome/web.dev/search?q=remark-frontmatter

Reply:
I think adding a markdown linter is a great idea. However, it feels a bit out of scope for this PR.

Reply:
Noted. Do you mind filing a feature request to keep track of it?